src/python/pants/backend/go/module.py

```diff
@@ -79,6 +80,9 @@ def basic_parse_go_mod(raw_text: bytes) -> Tuple[Optional[str], Optional[str]]:
 
 # Parse the output of `go mod download` into a list of module descriptors.
 def parse_module_descriptors(raw_json: bytes) -> List[ModuleDescriptor]:
+    if len(raw_json) == 0:
+        return []
```

Comment:
Is this to handle an error case, or is it an optimization? If the latter, would drop it probably; if the former, would add a comment.

Reply:
Error case. The `ijson` decoder could not deal with blank input (at least not with the options I've used thus far; might bear some research).

Reply:
Done.